index.rst

```diff
@@ -302,107 +305,153 @@ Is this case, we could have something like:
+
+    - See this :download:`example _init.yaml <_static/_init.yaml>` for the ATSpectrograph CSC.
+    - This file is the first configuration file loaded by the CSC
+    - It is not possible to load this file using the `configurationToApply` attribute
+    - Note that all CSCs having multiple algorithms, each with different required attributes, must have an initial set of defaults in this file.
+
+#.  Also in the configuration repository for the given CSC, there shall be a file corresponding to each site where the CSC is used (e.g. ``_summit.yaml, _ncsa.yaml, _base.yaml``).
+    These files contain site specific configuration parameters such as IP addresses and ports.
+    SalObj determines what site file should be loaded automatically by parsing the ``LSST_DDS_PARTITION_PREFIX`` environment variable
+
+    - See this :download:`example _summit.yaml <_static/_summit.yaml>` for the ATSpectrograph CSC.
+    - This file is the second configuration file to get loaded by the CSC and can override any previously declared values.
+    - It is not possible to load this file using the `configurationToApply` attribute
+    - Between this file and the ``_init.yaml`` file, **the configuration must be fully defined**.
 
-#.  On the configuration repository there shall be a ``_base.yaml`` file defining all the base configuration values (we use "base" instead of "default").
+#.  An additional configuration file provides overrides for the configuration parameters set by the previous files.
 
-    - See this :download:`example _base.yaml <_static/_base.yaml>` for the ATSpectrograph CSC.
+    - See this :download:`configuration parameter override example file <_static/ATSpectrograph_example_config.yaml>` for the ATSpectrograph CSC.
+    - This file is the third configuration file to get loaded by the CSC and can override any previously declared values.
+    - These files are loaded using the `configurationToApply` attribute in the ``start`` command
+    - These are not expected to be required as part of regular operations and are meant to be used when a non-standard configuration is required
+    - If an override configuration file is also site specific, then a prefix should be added indicating which site it belongs with (e.g. ``summit_reduced_stage_travel.yaml``)
 
 #.  The labels file (e.g. ``_labels.yaml``) shall continue to exist with the same format and purpose.
-#.  Additional configuration files can provide new values for individual configuration parameters.
 
-    - See this :download:`example configuration file <_static/ATSpectrograph_example_config.yaml>` for the ATSpectrograph CSC.
+        - No label shall exist having the name ``default``.
```

Comment:
we are proposing to remove labels so , maybe remove this phrase and start with invalid file names?